packages/fluentui/react-bindings/src/hooks/useFrameFocus.ts

```diff
@@ -0,0 +1,7 @@
+import { useFrameFocusDispatch } from './useFrameFocusDispatch';
+import { useFrameListener } from './useFrameListener';
+
+export const useFrameFocus = (enableFrameFocusDispatch: boolean, $document: Document, cb: (e: Event) => void) => {
```

Comment:
nit:
```suggestion
export const useOnFrameFocus = (enableFrameFocusDispatch: boolean, $document: Document, cb: (e: Event) => void) => {
```

Reply:
we should be detailled with jsdoc in these hooks, this is an edge case that might not be so obvious at first look